tests/integration/test_charm.py

```diff
@@ -84,3 +84,47 @@ async def test_connect_client_to_server(ops_test: OpsTest):
     assert (
         ret_code == 0
     ), f"Test returned code {ret_code} with stdout:\n{stdout}\nstderr:\n{stderr}"
+
+
+async def test_connect_to_console(ops_test: OpsTest):
+    """
+    Tests a deployed MinIO app by trying to connect to the MinIO console
+    """
+    # TODO: This presumes we're using microk8s.  Fix that.
+    #  could just call kubectl and count on the outside environment aliasing it for us?
+    #  or could pass kubectl path in as a variable?
+
+    application = ops_test.model.applications[APP_NAME]
+    config = await application.get_config()
+    port = config["console-port"]["value"]
+    # TODO: how do I dynamically retrieve the minio k8s service name?
```

Comment:
App name should be enough.

Reply:
I will remove this comment from both tests